build-logic/convention/src/main/java/AndroidDependencyInjectionConventionPlugin.kt

```diff
@@ -0,0 +1,15 @@
+import com.nibalk.tasky.buildlogic.convention.libs
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.dependencies
+
+class AndroidDependencyInjectionConventionPlugin: Plugin<Project> {
```

Comment:
What does this configure? It just applies a bundle dependency, so why not apply that right away instead of having to do this via a separate plugin?
Convention plugins are great to bundle related Gradle logic and different types of dependencies, but if they just add one line of code you might as well just add it to each Gradle file individually.

Reply:
I initially added this Plugin class to manage dependencies for Dagger/Hilt, but then switched it to Koin. :)
I'll revert these changes back to each Gradle file in my next PR.